ee/be/modules/350-node-local-dns/monitoring/grafana-dashboards/kubernetes-cluster/dns-node-local.json

```
@@ -13,14 +16,18 @@
     ]
   },
   "description": "",
-  "editable": false,
-  "gnetId": null,
+  "editable": true,
```

All imbedded dashboards must me not editable.
```suggestion
  "editable": false,
```